src/App.tsx

```diff
@@ -7,6 +9,7 @@ function App() {
     <h1 className="text-3xl font-bold underline">
       Hello world! Test deploy!
```

Comment:
**TODO:** You can remove this `<h1>` with Hello World! Test Deploy! in it now, as this was only used for testing.